jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/MavenScmEnricherTest.java

```diff
@@ -78,27 +68,18 @@ public void testMavenScmOnlyConnection() {
             .scmUrl("scm:git:git://github.com/jkubeio/kubernetes-maven-plugin.git")
             .build();
         // Setup mock behaviour
-        new Expectations() {
-            {
-                {
-                    context.getProject();
-                    result = project;
-                }
-            }
-        };
-
+        when(context.getProject()).thenReturn(project);
         MavenScmEnricher mavenScmEnricher = new MavenScmEnricher(context);
 
         KubernetesListBuilder builder = new KubernetesListBuilder().withItems(new DeploymentBuilder().withNewMetadata().withName("foo").endMetadata().build());
 
         mavenScmEnricher.create(PlatformMode.kubernetes, builder);
 
         Map<String, String> scmAnnotations = builder.buildFirstItem().getMetadata().getAnnotations();
-        assertNotNull(scmAnnotations);
 
-        Assert.assertEquals(1, scmAnnotations.size());
-        Assert.assertFalse(scmAnnotations.containsKey(JKubeAnnotations.SCM_TAG.value()));
-        Assert.assertEquals("scm:git:git://github.com/jkubeio/kubernetes-maven-plugin.git", scmAnnotations.get(JKubeAnnotations.SCM_URL.value()));
+        assertThat(scmAnnotations).isNotNull().hasSize(1)
+                .containsEntry(JKubeAnnotations.SCM_URL.value(),"scm:git:git://github.com/jkubeio/kubernetes-maven-plugin.git");
+        assertThat(scmAnnotations.containsKey(JKubeAnnotations.SCM_TAG.value())).isFalse();
```

Comment:
These two assertions can be chained together:
```suggestion
        assertThat(scmAnnotations).isNotNull().hasSize(1)
            .containsEntry(JKubeAnnotations.SCM_URL.value(),"scm:git:git://github.com/jkubeio/kubernetes-maven-plugin.git")
            .doesNotContainKey(JKubeAnnotations.SCM_TAG.value());
```